Gruntfile.js

```diff
@@ -186,13 +194,12 @@ module.exports = (grunt) => {
         },
         jsbeautifier: {
             default: {
-                src: ['src/**/*.js', 'index.js']
+                src: ['src/**/*.js', 'index.js'],
+                options: beautifierOptions
             },
             'git-pre-commit': {
                 src: ['src/**/*.js', 'index.js'],
-                options: {
-                    mode: 'VERIFY_ONLY'
-                }
+                options: Object.assign({ mode: 'VERIFY_ONLY' }, beautifierOptions)
```

Comment:
this hash of beautifier options is used elsewhere; I'm mixing it in with the pre-existing option here. (btw, as Sim has pointed out, this grunt sub-task appears to be in disuse.)